pages/record/_.vue

```diff
@@ -9,32 +9,67 @@
     data-qa="record page"
     class="mt-5"
   >
-    <b-row class="mb-3 mx-0 card p-3">
+    <b-row class="mb-3">
       <b-col
         cols="12"
-        class="mb-3 px-0"
+        lg="9"
       >
-        <MediaPresentation
-          v-if="selectedMedia"
-          :codec-name="selectedMedia.edmCodecName"
-          :image-link="image.link"
-          :image-src="image.src"
-          :mime-type="selectedMedia.ebucoreHasMimeType"
-          :url="selectedMedia.about"
-          :width="selectedMedia.ebucoreWidth"
-          :height="selectedMedia.ebucoreHeight"
-        />
+        <div class="card p-3">
+          <div
+            class="card-grid"
+            :class="isRichMedia && 'card-grid-richmedia'"
+          >
+            <div class="card-heading">
+              <h1
+                v-if="titlesInCurrentLanguage.mainTitle"
+                :lang="titlesInCurrentLanguage.mainTitle.code"
+              >
+                {{ titlesInCurrentLanguage.mainTitle.value }}
+              </h1>
+              <!-- eslint-disable vue/no-v-html -->
+              <p
+                v-if="titlesInCurrentLanguage.altTitle"
+                :lang="titlesInCurrentLanguage.altTitle.code"
+                class="font-weight-bold"
+                v-html="titlesInCurrentLanguage.altTitle.value"
```

Comment:
I don't think this needs to be in v-html. It should just be the text content of the element.

Reply:
This has been updated with the rework on the headings.